test/iaux-sharing-options.test.ts

```diff
@@ -38,23 +38,13 @@ describe('<iaux-in-share-panel>', () => {
 
     await el.updateComplete;
 
-    el.sharingOptions.forEach(option => {
-      const button =
-        el.shadowRoot && el.shadowRoot.querySelector(`a.${option.class}`);
+    el.sharingOptions.forEach((option, i) => {
+      const button = el.shadowRoot?.querySelectorAll('a')[i];
       expect(button).to.exist;
       expect(button?.getAttribute('href')).to.equal(option.url);
     });
   });
 
-  it('toggles visibility of embed options', async () => {
```

Comment:
hi, did we not want this?

Reply:
I switched to using the html details tag, so was able to delete our code managing the visibility!